tango/integrations/beaker/workspace.py

```diff
@@ -292,7 +295,10 @@ def _get_run_from_dataset(self, dataset: Dataset) -> Optional[Run]:
 
         import concurrent.futures
 
-        with concurrent.futures.ThreadPoolExecutor() as executor:
+        with concurrent.futures.ThreadPoolExecutor(
+            max_workers=50,
```

Comment:
Actually this probably makes the situation worse. The default is not more than 36.
https://github.com/python/cpython/blob/e929dae42ad6aaf55e5cce6339c1ef97c01c8f7b/Lib/concurrent/futures/thread.py#L142

Reply:
Oh. Then the documentation for that is wrong.

Reply:
Yeup 🙃